content/english/vegan-culture/movies-worth-watching.md

```diff
@@ -0,0 +1,79 @@
+A major reason why meat-eating is still the dominant culture is because corporations romanticize their products. Think KFC, McDonald’s and Burger King. It takes slaughterhouse videos and undercover investigations for us to see how our food is produced. Consumers, having no real knowledge of the cruelty they fund, continue to consume these products, without any guilt. Okja comments on this beautifully. The Superpig contest is nothing but a PR exercise intended on boosting sales and painting the people at Mirando as the good guys. The superpigs are marketed to be organic, non-GMO and better for the environment. Mirando even hires a celebrity zoologist and veterinarian Johnny Wilcox, played by Jake Gyllenhaal, to be the face of their brand. As the film progresses, we are taken behind the closed doors and made aware of the many facts that are being conveniently left out. Obviously, they are not pleasant." *- Wayne D'Mello ([Netflix Movie Okja Gets The Animal Rights Movement Perfectly](http://web.archive.org/web/20190806132813/https://www.filmcompanion.in/4-reasons-why-netflix-movie-okja-gets-the-animal-rights-movement-perfectly/))*.  
+**links:** [Youtube trailer](https://www.youtube.com/watch?v=AjCebKn4iic), [Netflix](https://www.netflix.com/nz/title/80091936). 
+
+## Dark City
+**By:** Alex Proyas (Director & Writer)    
+**Country or Origin:**  United States   
+**Genre:** Mystery, Sci-fi, Thriller.   
+**Release Date:** 1998   
+**Issues covered:**  Vivisection, animal rights, [human superiority]({{< ref "/anti-vegan/humans-are-superior" >}}).  
+**Thoughts:** A dark, brooding, Cartesian nightmare where everything is filtered through a veil of doubt. Primarily aimed at the philosophical question 'are we more than the sum of our memories'. However, Dark City is also a useful thought experiment on vivisection and human justification for vivisection. *Gregory Sadler* notes that the experimenters are “'antagonists' rather than 'villains'. There is a reason for this – despite appearances, they are not actually evil, merely misguided. They have what seem to them perfectly good reasons for what they do, and they seem genuinely frightened and disturbed when their plans are disrupted... *- ([Dark City, Descartes, and Narrative Doubts](http://web.archive.org/web/20151002092115/https://gbsadler.blogspot.com/2011/09/dark-city-descartes-and-narrative.html))*.  Pay close attention to the human doctor (especially the mouse in the maze) and the film's final scenes.     
```

Comment:
Cartesian? Sorry, not one I know! Probably doesn't matter as it's clear enough what the sentence is on about.

Reply:
After Rene Descartes. The consideration that you can doubt everything. Hoe do you know you exist, your memories are true, your every move isn't controlled by an evil demon.  like you say think its clear enough so will leave (and it will give folks something to google ...